internal/cli/atlas/deployments/search/indexes/describe_test.go

```diff
@@ -111,6 +112,7 @@ func TestDescribe_RunLocal(t *testing.T) {
 		CollectionName: "coll",
 		Database:       "db",
 		Status:         pointer.GetStringPointerIfNotEmpty(expectedStatus),
+		Type:           pointer.GetStringPointerIfNotEmpty(expectedType),
```

Comment:
```suggestion
		Status:         &expectedStatus,
		Type:           &expectedType,
```

Reply:
same as above, but here we can convert this to var (as scope is limited to this test), is that the desired approach for consistency?